composer.json

```diff
@@ -45,6 +45,14 @@
         "spatie/phpunit-watcher": "^1.23",
         "vimeo/psalm": "^5.20"
     },
+    "suggest": {
+        "yiisoft/db-sqlite": "For using with SQLite",
+        "yiisoft/db-mysql": "For using with MySQL",
+        "yiisoft/db-pgsql": "For using with PosgtgreSQL",
+        "yiisoft/db-mssql": "For using with Microsoft SQL Server",
+        "yiisoft/db-oracle": "For using with Oracle",
```

Comment:
Wouldn't it better mention the only `yiisoft/db`?

Reply:
I don't feel like changing codes. At the moment I'm standardizing all the packages, maybe you've already noticed this.... 🤭 but I think Alexander confirmed to only put the ...db.

Reply:
sorry. I thought it was me 😕